src/coreclr/jit/codegenlinear.cpp

```diff
@@ -847,6 +847,13 @@ void CodeGen::genCodeForBBlist()
         }
 #endif // defined(DEBUG) && defined(USING_VARIABLE_LIVE_RANGE)
 
+#ifdef DEBUG
```

Comment:
Won't this assert always be `true` given that at the beginning of this method we do:

```c++
bool needLabel = (block->bbFlags & BBF_HAS_LABEL) != 0;
...
...
if (needLabel)
{
    // Mark a label and update the current set of live GC refs

    block->bbEmitCookie = GetEmitter()->emitAddLabel(gcInfo.gcVarPtrSetCur, gcInfo.gcRegGCrefSetCur,
                                                     gcInfo.gcRegByrefSetCur, false DEBUG_ARG(block));
}
```

Reply:
Unless something else could modify it, which I don't think that is happening. I put it there for a sanity check.